relayer/relays/beefy/scanner.go

```diff
@@ -129,6 +131,21 @@ func scanCommitments(ctx context.Context, api *gsrpc.SubstrateAPI, startBlock ui
 				return
 			}
 
+			isMandatory := false
```

Comment:
I would rather we go all the way and add a documented `MandatoryCommitmentsOnly` config setting. Otherwise this code is going to be forgotten about it.

Reply:
We may not really need it as can be already achieved by setting
https://github.com/Snowfork/snowbridge/blob/aab91ae8c8d99072deb8d8d3230306763eb46996/web/packages/test/config/beefy-relay.json#L8 with a value big enough(i.e. more than blocks in an epoch).

Reply:
Addressed in [a791b2a](https://github.com/Snowfork/snowbridge/pull/1138/commits/a791b2a7aef4bc3107710370a3a268a4166ac64c).

We can use the `update-period` but this is more explicit and is good for testing the mmr fix as well. Currently turning this setting on will fail untill we update from upstream polkadot-sdk. I tested this change with cherry-picking .
```
$ git cherry-pick e5bb11b008ca6c1b5a07cab94a2327538f3bf15c
```

I think this setting also works well with incentivizing beefy relayers. For instance, our relayer can run in a mode where it only sync's mandatory commitments (i.e. 1 commitment per epoch). And if there are pending messages and an incentive is provided by a user to speed up the transaction then it can sync the next commitment it sees.